dev-feb-2017/mawer-london.md

```diff
@@ -0,0 +1,9 @@
+#London, UK
+
```

Comment:
Can you provide your favorite pub or restaurant to get a real English breakfast?